src/elements/List/List.js

```diff
@@ -15,7 +15,12 @@ export default class List extends Component {
   };
 
   render() {
-    const classes = classNames('sd-list', 'ui', this.props.className, 'list');
+    const classes = classNames(
+      'sd-list',
+      'ui',
+      this.props.className,
+      'list'
+    );
```

Comment:
Tiny thing - fixed formatting issue on the `<List>` component - included in this PR